From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Tue, 11 Aug 2026 22:48:16 +0200
Subject: build: Bump dependency on setting schemas 51.beta

Since commit 8a00d2b55 we use the new keyboard-focus-visible-timeout key
that was introduced on the schemas 51.beta

Origin: https://gitlab.gnome.org/GNOME/gnome-control-center/-/merge_requests/3466
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 0b19251..3894854 100644
--- a/meson.build
+++ b/meson.build
@@ -233,7 +233,7 @@ gnome_bg_dep = dependency('gnome-bg-4')
 gnome_qr_gtk_dep = dependency('gnome-qr-gtk-4')
 gnome_settings_dep = dependency('gnome-settings-daemon', version: '>= 48.rc', fallback: 'gsd')
 goa_dep = dependency('goa-1.0', version: goa_req_version, fallback: 'goa')
-gsettings_desktop_dep = dependency('gsettings-desktop-schemas', version: '>= 51.alpha')
+gsettings_desktop_dep = dependency('gsettings-desktop-schemas', version: '>= 51.beta')
 libxml_dep = dependency('libxml-2.0')
 pulse_dep = dependency('libpulse', version: pulse_req_version)
 pulse_mainloop_dep = dependency('libpulse-mainloop-glib', version: pulse_req_version)
